crates/rpc/src/error.rs

```diff
@@ -192,6 +199,13 @@ impl From<BeerusRpcError> for ErrorObjectOwned {
             BeerusRpcError::Other(other_err) => {
                 ErrorObjectOwned::owned(other_err.0, other_err.1, None::<()>)
             }
+            BeerusRpcError::UnexpectedSpecVersion(err) => {
```

Comment:
This needs significant refactoring of errors project-wise, it would be way out of scope for this PR.